### DISCUSSION TITLE
Editing of automatic actions

### DISCUSSION THREAD

**Discussion by QuantumQuacken** [aljawaid/AutomaticActionUX#18] [Q&A] [answered]
Apologies if I missed the way to do it, but I expected, when opening the cogwheel menu for a single action, to be able to edit it ([screenshot for reference](https://imgur.com/a/UWGfMo6)).
Is this a missing feature, a bug, or just the way it is because the action can only be deleted?

@aljawaid I'd also like to thank you for the amazing plugins you've developed and continue to maintain!

**Comment by aljawaid** [comment, marked as answer]
@QuantumQuacken Hello, thanks for the message.  The edit action feature is not available due to the way Kanboard is coded in the core.  It is too complicated to edit an action without major changes in the original code so therefore you can only delete the action to recreate it.  Not much I can do on that part sorry.

Thanks for the feedback on the plugins!  Your kind words motivate me a great deal. Feel free to buy me a coffee or donate to any of the plugin repositories to contribute towards the ever-growing maintenance costs.  The links are in each repository on the right hand side.  If you haven't already done so, I suggest you install PluginManager as a must... that will help you choose you the right plugins for your needs.

**Reply by QuantumQuacken** [reply]
Hello! I thought that was the case, but I don't have any knowledge of Kanboard's core code, so I couldn't be sure.
Will definitely contribute once I get back to a safer financial state!

Yes! I'm already using PluginManager and it's fantastic. I will post later on that repository as I have found a minor issue, so see you there :)
Thank you!

**Reply by aljawaid** [reply]
@QuantumQuacken Let me know in PluginManager
